### ISSUE TITLE
Creation time in record view

### ISSUE THREAD

**Issue by ACharbonneau** [nih-cfde/cfde-deriva#60]
Not clear to the user what "creation time" refers to...sample collection date? sample processing date? ingest date? If it's about the database ingest, it should just not show in the records view by default. If it's about a scientifically meaningful value, it needs more explanation
#57

**Comment by karlcz** [comment]
Yes, that is the `creation_time` field of the C2M2 model, used across many of the entity types.  We should review documentation of this field and (for consistency) the encoding of it in the frictionless schema comment which eventually populates the GUI tooltip.
